### PR TITLE
Added the right system-ui font config for grommet theme.

### PR DESCRIPTION
**Pull request by alansouzati**
#### What does this PR do?
Changes Grommet theme to properly definitely all system fonts across different devices
#### Where should the reviewer start?
base.js
#### What testing has been done on this PR?
manual
#### How should this be manually tested?
open storybook
#### Do the grommet docs need to be updated?
no
#### Should this PR be mentioned in the release notes?
no
#### Is this change backwards compatible or is it a breaking change?
backwards

### CODE REVIEW

File: src/js/themes/grommet.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ export const grommet = deepFreeze({
       background: '#ffffff',
     },
     font: {
-      family: '"San Francisco", "Helvetica Neue", Helvetica, Arial, sans-serif',
+      family:
+        '-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Fira Sans", "Droid Sans",  "Helvetica Neue", Arial, sans-serif,  "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";',
```
Comment (ericsoderberghp) [inline src/js/themes/grommet.js]: Can you articulate where this list came from? These are not the fonts @L0ZZI wanted to promote for the `grommet` theme.

Comment (alansouzati) [reply src/js/themes/grommet.js]: All the fonts are contained on there it is a super set. This lists comes from github and atlassian. If we want to do system fonts this is the way to go

Comment (ericsoderberghp) [reply src/js/themes/grommet.js]: Let's discuss with @L0ZZI tomorrow then.

Comment (L0ZZI) [reply src/js/themes/grommet.js]: I do like this stack. It's a little more inclusive to make a better display on typography respective systems. I'm in support of it. Any cost to the change?

Comment (alansouzati) [reply src/js/themes/grommet.js]: no it is a super set. we should be fine.

**Pull request merged**